qiime2/core/testing/format.py

```diff
@@ -98,3 +98,19 @@ def single_ints_path_maker(self, num):
             return 'nested/file%d.txt' % num
         else:
             return 'file%d.txt' % num
+
+
+class UnimportableFormat(TextFileFormat):
+    """
+    Useless format which maps to nothing, used for testing the plugin
```

Comment:
This comment isn't entirely accurate since there's a transformer mapping it to *something* -- I'd be fine with just removing the docstring now that the format name is self-explanatory.